CHANGELOG.md

```diff
@@ -9,6 +9,9 @@
     - `-blocks-storage.bucket-store.ignore-blocks-within` now defaults to `10h` (previously `0`)
     - `-querier.query-store-after` now defaults to `12h` (previously `0`)
     - `-querier.shuffle-sharding-ingesters-lookback-period` now defaults to `13h` (previously `0`)
+* [CHANGE] The following settings are now classified as advanced because the defaults should work for most users and tuning them requires in-depth knowledge of how the read path works: #TBD
```

Comment:
```suggestion
* [CHANGE] The following settings are now classified as advanced because the defaults should work for most users and tuning them requires in-depth knowledge of how the read path works: #1929
```